### ISSUE TITLE
Getting empty list of lists when using expand_param_grid on forest models

### ISSUE THREAD

**Issue by robme-l** [online-ml/river#1356]
As mentioned in the title, when using `utils.expand_param_grid` on any of the `forest` classifiers (namely `AMFClassifier` and `ARFClassifier`), I get an empty list of lists such as the following `[[],[],...]`.

**Comment by robme-l** [comment]
Following this up: after some playing around, I noticed the model's representation comes back as an empty list for `forest` models. However, under the hood they are actually initialized correctly. How do I know? Using each model's `_get_params()` method and attempting to train these seemingly "empty" models anyways.

So to reaffirm, the problem is not with `utils.expand_param_grid`, it lies with the string/HTML representation with `forest` models. Tagging @MaxHalford here to bring this to your attention. I haven't checked the `forest` source code yet but this was my experience. Closing this issue since it's not related to  `utils.expand_param_grid`.
